rca/project_styleguide/templates/patterns/pages/programmes/programme_index.html

```diff
@@ -20,6 +20,17 @@ <h1 class="title-area__header heading heading--display-two">{{ title_multiline|l
         </header>
 
         <div class="page__content">
+            {{ filters|json_script:"programme-listing-filters" }}
+            <div class="filters">
+                {% for f in filters %}
+                    <ul class="filters filters__list">
+                        {{ f.title }}
+                        {% for i in f.items %}
+                            <li class="filters filters__list item">{{ i }}</li>
+                        {% endfor %}
+                    </ul>
+                {% endfor %}
+            </div>
```

Comment:
@thibaudcolas feel free to remove this markup, just there for testing purposes really and as an example

Reply:
👍 good idea